### DISCUSSION TITLE
[FeatureRequest] Thresholds: Allow rounded values to affect the thresholds

### DISCUSSION THREAD

**Discussion by luanhdeandrade** [grafana/grafana#38194] [Dashboards, Panels & Visualizations]
**What would you like to be added**: An option to allow the threshold colors to be applied when the *rounded* value reaches the threshold limit. For example: 99.45192756482% when rounded up to 1 decimal digit becomes 99.5%.

Currently, the thresholds are not affected by the rounded number, so, following the above above, if I create a threshold value of 99.5%, my graph will round up 99.45192756482 to 99.5%, but the threshold color will not apply.

**Why is this needed**: Sometimes, some goals are not precise enough (because they don't need to) and this leads the dashboard to seem incoherent when you have a value in a panel (such as gauge) that is equals to your threshold (therefore, it is ok), but the color that is applied to it means that it is not ok.

Please, refer to this issue #29960 for a more practical example
Thanks!

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Reply by rodbloke** [reply]
This would be a useful option to have.  The rounding of values can be confusing for consumers of reports.

**Comment by nmarrs** [comment]
Thresholds are based off of actual values, and not rounded values. We suggest increasing the decimals shown to avoid this issue.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
